src/biogeochem/DUSTMod.F90

```diff
@@ -356,86 +583,201 @@ subroutine DustEmission (bounds, &
+            frc_thr_rghn_fct(p) = 0.0_r8            ! When LAI > 1, the drag partition effect is zero. dmleung 16 Feb 2024.
+         end if
+
+         !########## end of drag partition effect #######################################################
 
-            wnd_frc_slt = fv(p)
+         ! save soil friction velocity and roughness effect before the if-statement
+         wnd_frc_soil(p) = wnd_frc_slt  ! save soil friction velocity for CLM output, which has drag partition and Owen effect
+         ! 20 Feb 2024: dmleung notes that Leung does not consider the Owen's effect. This is Jasper Kok's decision. The Owen's effect should be in Zender's DEAD emission scheme.
+
+         ! save land mobile fraction
+         lnd_frc_mble(p) = lnd_frc_mbl(p)  ! save land mobile fraction first, before the if-statement
+         ! only perform the following calculations if lnd_frc_mbl is non-zero
+
+         if (lnd_frc_mbl(p) > 0.0_r8) then  ! if bare land fraction is larger than 0 then calculate the dust emission equation
+
+            ! reset these variables which will be updated in the following if-block
             flx_mss_hrz_slt_ttl = 0.0_r8
             flx_mss_vrt_dst_ttl(p) = 0.0_r8
 
             ! the following line comes from subr. dst_mbl
             ! purpose: threshold saltation wind speed
 
-            wnd_rfr_thr_slt = u10(p) * wnd_frc_thr_slt / fv(p)
+            wnd_rfr_thr_slt = u10(p) * wnd_frc_thr_slt / fv(p)     ! use if we want the default Z03 scheme; for Leung 2023 this is not needed
 
-            ! the following if-block comes from subr. wnd_frc_slt_get 
-            ! purpose: compute the saltating friction velocity
-            ! theory: saltation roughens the boundary layer, AKA "Owen's effect"
+            ! the following comes from subr. flx_mss_hrz_slt_ttl_Whi79_get
+            ! purpose: compute vertically integrated streamwise mass flux of particles
 
-            if (u10(p) >= wnd_rfr_thr_slt) then
-               wnd_rfr_dlt = u10(p) - wnd_rfr_thr_slt
-               wnd_frc_slt_dlt = 0.003_r8 * wnd_rfr_dlt * wnd_rfr_dlt
-               wnd_frc_slt = fv(p) + wnd_frc_slt_dlt
+            if (wnd_frc_slt > wnd_frc_thr_slt_it) then! if using Leung's scheme, use impact threshold for dust emission equation; if Zender, uses fluid threshold (wnd_frc_thr_slt) for dust emission equation
+
+               !################### for Leung et al. (2023) ################################################
+               !################ uncomment the below block if want to use Leung's scheme ###################
+
+               !flx_mss_vrt_dst_ttl(p) = dst_emiss_coeff(p) * mss_frc_cly_vld(c) * forc_rho(c) * ((wnd_frc_slt**2.0_r8 - wnd_frc_thr_slt_it**2.0_r8) / wnd_frc_thr_slt_std) * (wnd_frc_slt / wnd_frc_thr_slt_it)**frag_expt  ! Leung et al. (2022) uses Kok et al. (2014) dust emission euqation for emission flux
```

Comment:
@dmleung this commented out line only has one difference to the working code. Is there a reason to preserve the old? I'd like to delete it for readability.

Reply:
The uncommented code is the recently updated version. So, yes, please delete the older version! Thanks